### PR TITLE
move cmake finder config files to root dir

### PR DESCRIPTION
**Pull request by veronicaz41**
Fixes https://github.com/leapmotion/autowiring/issues/525

- [ ] only tested on Mac OSX, need to test on other platforms

### CODE REVIEW

**Comment by codemercenary** [comment]
I'll test on Windows.

**Comment by codemercenary** [comment]
Don't forget your labels.

File: CMakeLists.txt [hunk 1]
```diff
@@ -182,7 +182,7 @@ if(NOT AUTOWIRING_IS_EMBEDDED)
   install(FILES
     "${CMAKE_CURRENT_BINARY_DIR}/autowiring-config.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/autowiring-configVersion.cmake"
-    DESTINATION "cmake"
+    DESTINATION ${CMAKE_INSTALL_PREFIX}
```
Comment (codemercenary) [inline CMakeLists.txt]: Would `DESTINATION "."` be the same?

Comment (veronicaz41) [inline CMakeLists.txt]: No. ${CMAKE_INSTALL_PREFIX} is absolute path (default to /usr/local). "." is relative path, the config files will be installed to ${CMAKE_INSTALL_PREFIX}/./leapserial-config.cmake and ${CMAKE_INSTALL_PREFIX}/./leapserial-configVersion.cmake.

**Comment by jdonald** [comment]
+1

**Pull request merged**